teps/0003-tekton-catalog-organization.md

```diff
@@ -145,8 +145,8 @@ this](https://docs.google.com/document/d/1-czjvjfpuIqYKsfkvZ5RxIbtoFNLTEtOxaZB71
     3. Existing users of the resource can choose what versions of the
        resources they can use
 
-3. Updates to already published resources should be a new versioned
-   release instead of updating the existing released version.
+3. Updates to already published resources should be released as a new versioned 
```

Comment:
I think this may need some rephrasing,

```suggestion
If an update change the behavior of an already published tasks, the updated task should be a released as a new version.
```